src/openrct2-ui/windows/Options.cpp

```diff
@@ -1957,6 +1963,16 @@ static void window_options_text_input(rct_window *w, rct_widgetindex widgetIndex
         free(gConfigTwitch.channel);
         gConfigTwitch.channel = _strdup(text);
         config_save_default();
+    } else if (widgetIndex == WIDX_API_URL_BUTTON) {
+        if (strncmp("http://", text, 7) != 0 && strncmp("https://", text, 8) != 0) {
```

Comment:
This probably needs replacing with C++ equivalents.

Reply:
We use `String::Compare` for case-insensitive string comparison.

Reply:
`Equals`